src/azure-cli/azure/cli/command_modules/appservice/utils.py

```diff
@@ -223,6 +223,10 @@ def _normalize_location(cmd, location):
     return location
 
 
+def _normalize_stage_location(location):
```

Comment:
suggest moving this logic into "_normalize_location"

Reply:
Only if _normalize_location is only used for VNet integration flow ofcourse.

Reply:
since we only want to eliminate stage for vnet validation, suggest calling this method "_normalize_location_for_vnet_integration" or similar.  In the method, first call _normalize_location and then strip away "(stage)" and "stage".  This makes the code calling this function cleaner and easier to follow while encapsulating this odd behavior into a "vnet" specific helper function.